stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -57,11 +56,10 @@ class ToCPlaceholder(NamedTuple):
 
 class SubsetMap:
     def __init__(self, identities: list[int]) -> None: ...
-    def pick(self, unicode: int): ...
-    def dict(self): ...
+    def pick(self, unicode: int) -> int | None: ...
```

Comment:
I think this always returns `int`. If `unicode` isn't in `self._map` when the method is called, it's inserted into `self._map` before `self._map.get(unicode)` is called. https://github.com/PyFPDF/fpdf2/blob/48f837963b7ae1956a0e5b23815063d23c1964f5/fpdf/fpdf.py#L246